src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -184,6 +184,10 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
             // When the form is being used for Batch Gift Entry, the Form Template JSON
             // uses the @wire service below to retrieve the Template using the Template Id
             // stored on the Batch.
+
+            // get the form template if in batch mode so the 
+            // donationFieldTemplateLabel gets set
+            GeFormService.getFormTemplate();
```

Comment:
@psadie-sfdo @kenlewis-sfdo @bdvorachek @dfuller114 @voduyemi - I'm reviewing Vlad's work here and just curious about this call to GeFormService.getFormTemplate() - when there's a second call to that (with an added big ole' promise resolution thrown in) about four lines below this. Would it be possible to unify these two calls and instead dynamically assign the function that's called in the ".then()" call? And if so ... should we? 

I realize that the real purpose of geFormRenderer has drifted somewhat, but it seems like you would need to access the form template to render the form no matter what the context, and if there's different actions to take AFTER you've loaded the template it would be cleaner to have one call with different behaviors in the "then()" block depending on batch / not batch. But team consensus DEFINITELY outweighs my opinion here!

Reply:
@ptewson-sfdo I refactored so now there's only one call to GeFormService.getFormTemplate().